### PR TITLE
Features/01 general left panel trung

### PR DESCRIPTION
**Pull request by trungh13**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -21,4 +21,30 @@
...
   -moz-box-sizing:border-box;
+}
+*,
+*:before,
+*:after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
```
Comment (Nizar-Rahme) [inline src/App.css]: These css rules are duplicates of the previous ones

File: src/components/Aside/index.js [hunk 2]
```diff
@@ -1,5 +1,94 @@
-import React from 'react';
+import React, { Component } from 'react';
+import './Aside.css';
+import { NavLink } from 'react-router-dom';
 
-const Aside = () => <div>Aside</div>;
+class Aside extends Component {
+  state = {
+    loggedIn: true,
+  };
```
Comment (Nizar-Rahme) [inline src/components/Aside/index.js]: I think `loggedIn` should come as a `prop` and it's better to store it in Redux, so this component can be a stateless one

File: src/components/Aside/index.js [hunk 3]
```diff
@@ -1,5 +1,94 @@
...
+          <div className="divider" />
+          <NavLink to="/logout" className="waves-effect waves-orange">
+            Logout
+            <i className="material-icons">power_settings_new</i>
+          </NavLink>
+        </li>
+      </React.Fragment>
+    );
+  }
+}
```
Comment (Nizar-Rahme) [inline src/components/Aside/index.js]: It's better to break this component down into smaller components

**Comment by trungh13** [comment]
It's done :D

File: src/components/Aside/Aside.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  background-color: #fa9026 !important;
+}
+.switch label input[type='checkbox']:checked + .lever {
+  background-color: #fa9026 !important;
+}
+.switch label {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 9px;
+}
```
Comment (Nizar-Rahme) [inline src/components/Aside/Aside.css]: In general it's better to avoid tag-names to apply css rules

File: src/components/Aside/Aside.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      <li>
+        <div className="user-view">
+          <div className="background">
+            <img
+              src="https://s3.eu-west-2.amazonaws.com/integrify-system-assets/logo-black.png"
+              alt="integrify-logo"
+            />
+          </div>
+          <div className="user-info" href="#user">
+            <NavLink to="/profile" activeClassName="active" className="user-photo">
```
Comment (Nizar-Rahme) [inline src/components/Aside/Aside.js]: We could use Link instead of NavLink

File: src/components/Aside/Aside.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+            <div className="user-status">
+              <AsideSwitch present={auth.present} onChange={onChange} />
+            </div>
+          </div>
+        </div>
+      </li>
+      {renderLinks}
+      <li className="logout">
+        <div className="divider" />
+        <NavLink to="/api/user/logout" className="waves-effect waves-orange">
```
Comment (Nizar-Rahme) [inline src/components/Aside/Aside.js]: Link and NavLink don't make api requests

File: src/components/Aside/index.js [hunk 7]
```diff
@@ -1,5 +1,47 @@
...
+};
+
+const mapStateToProps = state => ({
+  auth: getAuth(state),
+});
+
+export default connect(
+  mapStateToProps,
+  { fetchUser: fetchUserAction, checkIn: checkInAction, checkOut: checkOutAction },
+)(Aside);
```
Comment (Nizar-Rahme) [inline src/components/Aside/index.js]: Let's move this file under containers directory

File: src/components/Aside/index.js [hunk 8]
```diff
@@ -1,5 +1,47 @@
...
-const Aside = () => <div>Aside</div>;
+import {
+  fetchUser as fetchUserAction,
+  checkIn as checkInAction,
+  checkOut as checkOutAction,
+} from '../../store/actions/index';
+import { getAuth } from '../../store/reducers';
 
-export default Aside;
+import './Aside.css';
```
Comment (Nizar-Rahme) [inline src/components/Aside/index.js]: Apply css styles inside the presentational component

File: src/components/Aside/Aside.css (hunk 4 above)
Comment (trungh13) [reply src/components/Aside/Aside.css]: will reconsider next sprint if got problem

**Pull request merged**